jbehave-support-core/src/main/java/org/jbehavesupport/core/ssh/SshHandler.java

```diff
@@ -216,4 +274,79 @@ protected String getSearchColumnName(ExamplesTable searchData) {
             throw new IllegalArgumentException("No search column found in example table");
         }
     }
+
+    private void storeInExtension(final MultiKey key, String logContent) {
```

Comment:
Nitpicking but what is our policy about final? Do we want to use it or dont? (and in this case I think it makes sense to use it either for both or for none)

Reply:
So it's none now.